test/utils/deepObject.test.js

```diff
@@ -0,0 +1,384 @@
+const { expect } = require('chai');
+const { toPath } = require('lodash');
+
+const logger = require('../../src/logger');
+const { deepObject } = require('../../src/utils');
+
+describe('[utils] deepObject', () => {
+	it('Should export all avaible actions', () => {
+		expect(deepObject.get).is.not.an('undefined');
+		expect(deepObject.pathsToArray).is.not.an('undefined');
+		expect(deepObject.getSimple).is.not.an('undefined');
+	});
+
+	describe('deepObject > pathToArray', () => {
+		const { pathsToArray, getSimple } = deepObject;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['getSimple' is assigned a value but never used. (no-unused-vars)](https://app.codacy.com/app/schul-cloud/schulcloud-server/pullRequest?prid=4646529)